rockcraft/oci.py

```diff
@@ -353,11 +353,14 @@ def set_default_user(self, user: str) -> None:
         _config_image(image_path, params)
         emit.progress(f"Default user set to {user}")
 
-    def set_entrypoint(self, entrypoint_service: str | None = None) -> None:
+    def set_entrypoint(self, entrypoint_service: str | None, build_base: str) -> None:
         """Set the OCI image entrypoint. It is always Pebble."""
         emit.progress("Configuring entrypoint...")
         image_path = self.path / self.image_name
-        entrypoint = [f"/{Pebble.PEBBLE_BINARY_PATH}", "enter", "--verbose"]
+        if build_base in ["ubuntu@20.04", "ubuntu@22.04"]:
```

Comment:
Can you just put the list without verbose by default and just append the version of base in 20.04 or 22.04?

Reply:
This will permit to remove the else and be more cleaner

Reply:
Done